### ISSUE TITLE
Can't swap user's positions in categories

### ISSUE THREAD

**Issue by Hsinto** [Bricklou/powercord-pindms#9]
In the BD version of the plugin, you can manually sort the dms within each category, I'm not sure if it's supposed to be in this plugin but it would be helpful.

**Comment by Bricklou** [comment]
I didn't implemented it in the UI on discord but it is possible to sort your DMs. You just have to edit the config manually.
I will try to look at this feature today.

**Comment by Hsinto** [comment]
Unfortunately it doesn't even seem like sorting by message last sent is currently working

**Comment by Hsinto** [comment]
You also can't organize dms from within discord client in the plugin settings

**Comment by Bricklou** [comment]
> Unfortunately it doesn't even seem like sorting by message last sent is currently working

Private Channel are sorted inside each categories depending on last message sent



> You also can't organize dms from within discord client in the plugin settings

Because I haven't implemented it, i'm currently working on a new way to render these categories, this shouldn't have any effect for user, but it should simplify a lot of things for me. This way I will be able to add the category sort feature.

**Comment by Hsinto** [comment]
> Private Channel are sorted inside each categories depending on last message sent

Yeah unfortunately this isn't working for me I think, they just seem to be arbitrarily sorted, as for the other config option, I see.

**Comment by Bricklou** [comment]
Friend list injection is a pain, and since the last update, I can't even say if I will be able to  continue to be able to make the injection properly
